### ISSUE TITLE
Island delete

### ISSUE THREAD

**Issue by Walabise** [BentoBoxWorld/BSkyBlock#516]
### Expected behavior

I would like to be able to delete an island from the console via _/bsbadmin delete <player>_

### Observed/Actual behavior

When I want to delete an island with the "bsbadmin delete" command from the console a return message tells me that the player does not have an island to create. Conversely, when I use the "bsbadmin delete" command directly in-game it works correctly. Has someone already had this problem ? No errors in the console.

### Steps/models to reproduce

/bsbadmin delete <player> in the console; The player has no island

### BentoBox version

Serveur PURPUR 1.20.4.
(git-Purpur-2162 (MC: 1.20.4))
Version de BentoBox : 2.3.0
Base de données : MARIADB
Mondes de jeu chargés :
bskyblock_world (BSkyBlock) : Overworld
Addons chargés :
Border 4.2.1 (ENABLED)
BSkyBlock 1.17.1

**Comment by tastybento** [comment]
Yes, I can replicate this. It works in game but not via the console. It's due to recent changes to enable multiple islands per player. Anyway, it's on the list for fixing!
